CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 
 - Links to ideas in admin digest emails work again
 - Votes statistics not showing up in the dashboard for some admins and project moderators.
+- Various parts of the sign-up flow were broken: all situations where the custom fields module was disabled, or where there were no custom fields activated by the admin. Certain cases where a user would refresh the page during the sign up flow would also lead to some strange sign-up limbo where the user was signed in but unable do to anything. These problems have been solved.
```

Comment:
Suggestion for alternative:

Fixes to sign-up flow: Disabling custom fields module, or admin not activating any custom fields, no longer breaks sign-up. Refreshing page at certain points in sign-up flow no longer creates an unregistered user.

Reply:
You can even go further. The changelog is for non-technical people, so "disabling a module" should probably not be in there.

Reply:
v2:

Fixes to sign-up flow: Not activating any custom registration fields no longer breaks sign-up. Refreshing page at certain points in sign-up flow no longer creates an unregistered user.